torch/quantization/_quantize_script.py

```diff
@@ -69,17 +70,17 @@ def prepare_script(model, qconfig_dict, inplace=False):
     _check_is_script_module(model)
     if not inplace:
         model = model.copy()
```

Comment:
That copy above seems redundant then.

Reply:
yeah, I'll do another refactor to remove `inplace` argument